setup.py

```diff
@@ -47,6 +47,19 @@ def read(fname):
     return open(os.path.join(os.path.dirname(__file__), fname)).read()
 
 
+try:
+    import django
+    os.environ.setdefault(
+        "DJANGO_SETTINGS_MODULE",
+        "analytical.tests.settings"
+    )
+    django.setup()
+except ImportError:
+    print(
+        "Could not import django."
+        "This is fine, unless you intend to run unit tests"
```

Comment:
There's at least a blank missing between the two sentences. They will stick together otherwise. Maybe also a final period could be added.